### PR TITLE
Grouping and aggregations blog post

### PR DESCRIPTION
**Pull request by systay**
Cross-posting from PlanetScale

### CODE REVIEW

File: content/en/blog/2024-09-05-grouping-and-aggregations-on-vitess.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+author: 'Andrés Taylor'
+date: 2024-07-22
```
Comment (deepthi) [inline content/en/blog/2024-09-05-grouping-and-aggregations-on-vitess.md]: This date needs to change.

**Review by deepthi — COMMENTED** [review]
Ideally we'd space these out by a week and publish them separately.
In any case, the dates need to be changed because we published another post on September 6.
If you could split these into 2 PRs, and put dates like Sep 13 and Sep 20 on them, I can approve them quickly. We can even merge them right away, and they will automatically show up on the website with the first deployment on or after the date on the post.

**Comment by systay** [comment]
@deepthi finally I did it. This PR has been updated, and I moved the other blog post to https://github.com/vitessio/website/pull/1876

File: content/en/blog/2024-10-29-grouping-and-aggregations-on-vitess.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
+---
+author: 'Andrés Taylor'
+date: 2024-10-29
+slug: '2024-10-29-grouping-and-aggregations-on-vitess'
```
Comment (deepthi) [inline content/en/blog/2024-10-29-grouping-and-aggregations-on-vitess.md]:
```suggestion
date: 2024-11-12
slug: '2024-11-12-grouping-and-aggregations-on-vitess'
```

**Review by deepthi — COMMENTED** [review]
Let's schedule it for 2 weeks from now, so that this goes _after_ the query planning blog post instead of before.

**Review by deepthi — APPROVED** [review]

**Pull request merged**